include/f_core/net/device/c_lora.h

```diff
@@ -0,0 +1,81 @@
+#ifndef C_LORA_H
+#define C_LORA_H
+
+#include <zephyr/drivers/lora.h>
+
+class CLora {
```

Comment:
Yea that was the intention. Need to figure out a clean way of doing it though since the underlying function calls have different parameter expectations that its difficult to standardize.